third_party/amd/lib/TritonAMDGPUTransforms/ReorderInstructions.cpp

```diff
@@ -58,7 +53,7 @@ findEarlyInsertionPoint(Block *block, Operation *move) {
       if (isa<triton::AtomicRMWOp, triton::AtomicCASOp>(wop))
         ipnt = bi;
       // Break at barrier
-      if (isa<gpu::BarrierOp>(wop))
+      if (isa<mlir::gpu::BarrierOp>(wop))
```

Comment:
This should be unnecessary since it built before.

Reply:
This happened because of the include of `"third_party/amd/lib/TritonAMDGPUToLLVM/Utility.h"`. In its turn, it include `"triton/Conversion/TritonGPUToLLVM/Utility.h"` which has the following definition

https://github.com/triton-lang/triton/blob/e2dc77b87be71f4ccb7ad6d3627b5be24407b6b2/include/triton/Conversion/TritonGPUToLLVM/Utility.h#L210-L217

As you can see it is `gpu` namespace. So, we need to be explicit about the namespace in `ReorderInstructions.cpp` file. Otherwise, we have a compilation error.